### DISCUSSION TITLE
Not getting any responses from the "Business Development team"

### DISCUSSION THREAD

**Discussion by brolifen** [lithnet/access-manager#187] [General]
In the past few months, I have sent a couple inquiries on being a possible partner so we can potentially roll out access manager to our customer base but the only (automatic) response I received so far was:
"Your request (xxx) has been received and is being reviewed by our Business Development team."
The inquiries had numbers:
464
526

Is there some backlog on the business side? I would understand if you guys had a sudden exponential growth too. We really like to work with this solution as its simplicity has great value.

**Comment by ryannewington** [comment]
@brolifen 

I can see that we have responded, and followed up on our end to both of those requests. Could the responses have gone to spam perhaps? You may need to add support@lithnet.zendesk.com to your allow list?

Tagging @ScottHuman who will try and contact you directly on this.
